app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  VALIDATA_GMAIL = /\A[a-zA-z0-9]+@[a-zA-z0-9]+\.[a-zA-z0-9]+(\.[a-zA-z0-9]+)*\z/
+  validate :name, presence: true, length: { maximun: 50 }
+  validate :password_digest, presence: true, length: { minimun: 150 }
```

Comment:
```suggestion
  validate :password_digest, presence: true, length: {minimun: 150}
```
chỗ khác sửa tương tự